java/configuration/src/main/java/sleeper/configuration/properties/instance/InstanceProperty.java

```diff
@@ -35,6 +35,8 @@ static Optional<InstanceProperty> getByName(String propertyName) {
         return Index.INSTANCE.getByName(propertyName);
     }
 
+    InstanceProperty getDefaultProperty();
```

Comment:
I think this getter isn't really needed. It's only used in UserDefinedInstancePropertyImpl.computeValue, which can access the field directly.

Reply:
Addressed